target/linux/pistachio/patches-4.4/801-spi-spidev-Increase-default-bufsize-to-64KB.patch

```diff
@@ -0,0 +1,26 @@
+From 877752e320586e21706b1de19945489378bb528a Mon Sep 17 00:00:00 2001
+From: Francois Berder <francois.berder@imgtec.com>
+Date: Fri, 13 Jan 2017 17:48:43 +0000
+Subject: [PATCH] spi: spidev: Increase default bufsize to 64KB
```

Comment:
It would be nice if this said why so that some looking at it would understand your reasons.